packages/bot-skeleton/src/scratch/blocks/Binary/Trade Definition/trade_definition_market.js

```diff
@@ -61,9 +62,33 @@ Blockly.Blocks.trade_definition_market = {
         // market options and Jump Diffusion Indices from
         // submarket options for Synthetic Indices
         // until multipliers are available for DBot
-        const market_options = active_symbols
-            .getMarketDropdownOptions()
-            .filter(option => option[1] !== 'cryptocurrency');
```

Comment:
maybe we should leave a const here instead of declaring let and move forex availability logic inside filter method